source/distributed/tria.cc

```diff
@@ -1830,6 +1830,8 @@ namespace parallel
       {
         std::string fname=std::string(filename)+".info";
         std::ifstream f(fname.c_str());
+        Assert (f, ExcMessage("Could not open the file <" +
+                              fname + ">."));
```

Comment:
I think this should have been an `AssertThrow` because it can also happen in release mode.

We've typically used `ExcIO` as the error message.